assets/crx/js/embed.js

```diff
@@ -82,4 +89,39 @@ function copyToClipboard(str, mimetype) {
 
     // Execute browser command 'Copy'
     document.execCommand("Copy", false, null);
-}
\ No newline at end of file
+}
+
+// Dialogs custom box
+var DialogController = null;
```

Comment:
Please camel-case this variable (`dialogController`) to conform with the project coding style. 👍